packages/ui/src/components/ResourceBlock/LinkMenuItem.tsx

```diff
@@ -40,7 +40,7 @@ const LinkMenuItem = ({
       <DropdownMenu.Content align="end" collisionPadding={2}>
         {links.map((link) => (
           <DropdownMenu.Item
-            key={link.title + link.href}
+            key={link.title + String(link.href)}
```

Comment:
Same comment about undfined.

Reply:
This one should never be undefined since it is required in the type.